<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_deck">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Deck</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DeckSource">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>insert source here</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_deck_name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Deck name here</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="drawn_card">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>card drawn</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cardName_search">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>card to search for</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_deck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>update deck</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="category_search">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>category to search for</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cardName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Program's card</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="probability">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>480</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>probability</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="conNewDeck">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>confirmation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="conUpdateDeck">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>confirmation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cardCount">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>card count</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuprobability">
    <property name="title">
     <string>probability</string>
    </property>
   </widget>
   <addaction name="menuprobability"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>add_deck</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>NewDeck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>update_deck</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>UpdateDeck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>CheckProbability()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>NewDeck()</slot>
  <slot>SelectDeck()</slot>
  <slot>UpdateDeck()</slot>
  <slot>CheckProbability()</slot>
  <slot>slot1()</slot>
 </slots>
</ui>
